samples/core/src/test/java/io/oasp/gastronomy/restaurant/offermanagement/batch/impl/productimport/ProductManagementTest.java

```diff
@@ -73,13 +77,13 @@ public void testFindProductsFiltered() {
     ProductSortBy sort = new ProductSortBy();
 
     List<ProductEto> products = this.offerManagement.findProductsFiltered(filter, sort);
-    assertEquals(products.size(), 6);
+    assertEquals(products.size(), NUMBER_PRODUCT_MEAL);
 
     filter.setFetchDrinks(true);
     filter.setFetchMeals(false);
     sort.setSortByEntry(ProductSortByHitEntry.DESCRIPTION);
     products = this.offerManagement.findProductsFiltered(filter, sort);
-    assertEquals(products.size(), 4);
+    assertEquals(products.size(), NUMBER_PRODUCT_DRINK);

```

Comment:
I really like this PR and it is a nice improvement over the ugly hardcoded magic values. But still the problem here is the test itself. Such assertions are ultra evil. First of all ```assertEquals``` is discouraged in OASP at all as we are using AssertJ what is much better and we need to get rid of any regular JUnit assert as well as hamcrest asserts. The sample is what should (some day) be considered as best practice and shows others how to do it right. Here we are doing things utterly wrong. The main issue is not the assertion syntax, though. It is asserting that the size of the result of a query has an expected magic value. This is very bad testing style. Instead you should assert the content of the result set and verify that particular hits are contained and that all containing hits fulfill the requested criteria.
BTW: I am totally aware that all this mess was not introduced by this PR. It is just that I am seeing this now watching the diff. We can file a new issue for that and it should not prevent accepting and merging this PR.